### PR TITLE
cffi: Fix invalid syntax for slices in generated Python

### PR DESCRIPTION
**Pull request by corona10**
This CL makes valid syntax for slices cases.

Updates go-python/gopy#156.

### CODE REVIEW

**Comment by corona10** [comment]
@sbinet 
I patched #158, but Travis CI is failed for same reason with this commit.
(https://travis-ci.org/go-python/gopy/jobs/355447086)

I suggest that this commit should be reverted.
https://github.com/go-python/gopy/commit/ab145c0f89e81cda6d5f0958f1385d4ab5d2b481

**Comment by sbinet** [comment]
you're probably right.

I have filed go-python/gopy#159.

could you rebase on top of 7568e06?

File: _examples/lot/lot.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// Reference from: https://github.com/initialed85/golang_python_binding_research
```
Comment (sbinet) [inline _examples/lot/lot.go]: I would add a blank line between our license and the provenance of this test file.

File: _examples/lot/lot.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	SomeInt           int64
+	SomeFloat         float64
+	SomeBool          bool
+	SomeListOfStrings []string
+	SomeListOfInts    []int64
+	SomeListOfFloats  []float64
+	SomeListOfBools   []bool
+}
+
+// GetOneWithTheLot returns a struct with a public properties of different types
```
Comment (sbinet) [inline _examples/lot/lot.go]: `s/a public properties/exported fields/`

File: _examples/lot/lot.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+func GetOneWithTheLot() OneWithTheLot {
```
Comment (sbinet) [inline _examples/lot/lot.go]: `s/GetOneWithTheLot/New/`

File: _examples/lot/lot.go [hunk 4]
```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// Reference from: https://github.com/initialed85/golang_python_binding_research
+
+package lot
+
+type OneWithTheLot struct {
```
Comment (sbinet) [inline _examples/lot/lot.go]: `s/OneWithTheLot/Value/` or `Lot`.

**Comment by corona10** [comment]
@sbinet Thanks for your review, I will update this CL after full-time work.

**Comment by corona10** [comment]
@sbinet 
I updated the CL as you reviewed.
All tests are passed :)

**Comment by corona10** [comment]
@sbinet Updated! No problem!

**Pull request merged**